src/Xamarin.Android.Build.Tasks/Xamarin.Android.TargetFrameworkInference.targets

```diff
@@ -0,0 +1,19 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+	<PropertyGroup Condition="'$(BaseIntermediateOutputPath)' == ''">
+		<BaseIntermediateOutputPath>obj</BaseIntermediateOutputPath>
+	</PropertyGroup>
+	<PropertyGroup Condition=" $(IntermediateOutputPath) == '' ">
+		<IntermediateOutputPath Condition=" '$(PlatformName)' == 'AnyCPU' ">$(BaseIntermediateOutputPath)$(Configuration)\</IntermediateOutputPath>
+		<IntermediateOutputPath Condition=" '$(PlatformName)' != 'AnyCPU' ">$(BaseIntermediateOutputPath)$(PlatformName)\$(Configuration)\</IntermediateOutputPath>
+	</PropertyGroup>
+	<PropertyGroup Condition="'$(AppendTargetFrameworkToOutputPath)' == '' and '$(AndroidUseLatestPlatformSdk)' != 'true'">
+		<AppendTargetFrameworkToOutputPath>true</AppendTargetFrameworkToOutputPath>
+		<TargetFrameworkToOutputPath>$(TargetFrameworkIdentifier)$(TargetFrameworkVersion.TrimStart('v').Replace('.', ''))</TargetFrameworkToOutputPath>
+	</PropertyGroup>
+	<PropertyGroup Condition="'$(AppendTargetFrameworkToOutputPath)' == 'true'">
+		<OutputPath Condition="'$(OutputPath)' != '' and !HasTrailingSlash('$(OutputPath)')">$(OutputPath)\</OutputPath>
+		<OutputPath>$(OutputPath)$(TargetFrameworkToOutputPath)</OutputPath>
```

Comment:
I can understand wanting to update `$(IntermediateOutputPath)`, but *`$(OutputPath)`*?!

I find it hard to believe that changing `$(OutputPath)` won't break large swaths of the world, and as `$(AppendTargetFrameworkToOutputPath)` is default True, that's quite a risk.

Reply:
I think we can avoid the OutputPath for now since the IntermediateOutputPath should be the one that is causing the 99% of the issues with the generated artifacts.

I will remove it and update the PR.